modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -25,62 +23,75 @@ Carefully review the following consequences before opting in to migration:
 * Migration is a disruptive process. The Machine Config Operator (MCO) has to update kubelet, which means degrading each node as it rolls out the new MachineConfig to each machine.
 ====
 
-=== Using the web console to opt in to automatic CSI migration
+Alternatively, if you do not want to upgrade vSphere, you can proceed with an {product-title} upgrade by performing an administrator acknowledgment:
 
-==== Prerequisites
+[source, cli]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-<version>-api-removals-in-4.14":"true"}}' --type=merge <1>
+----
+<1> Where `<version>` is the {product-title} and associated Kubernetes versions that you are upgrading from: either `4.13-kube-1.27` or `4.12-kube-1.26`.
```

Comment:
The values are hardcoded and can't be replaced and hence there is no need of a note

Reply:
Right but since it's an OCP 4.14 doc page, maybe we should cover both upgrade scenarios?
If you're upgrading from 4.12
`4.12 command`
If you're upgrading from 4.13
`4.13 command`

Reply:
yeah I agree. I think it might be worth listing the keys and commands separately though, rather than just templating them because templating requires customers to be smart about replacing the values. 

I would prefer to have something that customer can simply copy and paste in terminal. Lets list 4.12 and 4.13 acks separately.

Reply:
Fixed.